packages/happo/src/cli.js

```diff
@@ -0,0 +1,91 @@
+const crypto = require('crypto');
+
+const commander = require('commander');
+const {
+  config,
+  saveResultsToFile,
+  constructUrl,
+} = require('happo-core');
+
+const { server, uploadLastResult } = require('happo-viewer');
+
+function logAndExitWithFailure(error) {
+  console.error(error);
+  process.exit(1);
+}
+
+function exitWithSuccess() {
+  process.exit(0);
+}
+
+function targetForName(name) {
+  const target = config.targets.find(t => t.name === name);
+  if (!target) {
+    logAndExitWithFailure(new Error(`Target ${name} not found.`));
+  }
+  return target;
+}
+
+commander.command('run [<target>]').action((targetName) => {
+  let resultPromise;
+  if (targetName) {
+    resultPromise = targetForName(targetName).run();
+  } else {
+    resultPromise = Promise.all(config.targets.map(target => target.run()))
+      .then(results => results.reduce((a, b) => a.merge(b)));
+  }
+
+  return resultPromise
+    .then(saveResultsToFile)
+    .then(exitWithSuccess)
+    .catch(logAndExitWithFailure);
+});
+
+commander.command('review').action(() => {
+  server.start().then(() => {
+    console.log(`=> ${constructUrl('/review')}`);
+  });
+});
+
+commander.command('debug <target>').action((target) => {
+  targetForName(target).debug();
+});
+
+commander.command('review-demo').action(() => {
+  // server.start().then(() => {
+  //   console.log(`=> ${constructUrl('/review-demo')}`);
+  // });
+});
```

Comment:
If this turns out to be a pain to maintain, we can throw it out and add it back later on. This has been useful when Joe and I have been iterating on/testing the in-browser diffing implementation, but we're not using it much now.